test/e2e/network/loadbalancer.go

```diff
@@ -670,6 +670,35 @@ var _ = common.SIGDescribe("LoadBalancers", func() {
 		if framework.ProviderIs("azure") {
 			ginkgo.By("switching back to interal type LoadBalancer, with static IP specified.")
 			internalStaticIP := "10.240.11.11"
```

Comment:
ditto, let's see if we can get of hardcoded values and make the test really portable